packages/casadi/patches/0001-fix-python-include-dir.patch

```diff
@@ -0,0 +1,37 @@
+From 6241d84b52702ad9535781e0a8f4eea130323fd4 Mon Sep 17 00:00:00 2001
+From: Agriya Khetarpal <74401230+agriyakhetarpal@users.noreply.github.com>
+Date: Mon, 15 Jul 2024 20:39:19 +0530
+Subject: [PATCH 1/4] Fix the location of the Python include directory
+
+CasADi's setup code used an incorrect location for the Python include
+directory via the use of self.include_dirs[0], which isn't guaranteed
+to return the correct interpreter used during compilation. This patch
+replaces it with sysconfig.get_path('include').
```

Comment:
Can we upstream this? Please open a PR and link it here if you think it's appropriate.

Reply:
Looks like there's a similar PR already, so I've commented on it there with this suggestion: casadi/casadi#3759